docker-compose.yml

```diff
@@ -19,55 +19,52 @@ services:
       - "8089:8089"
     links:
       - web
+      - watch
 
-  web:
+  python:
     build: .
+    command: /bin/true
     volumes:
       - .:/src
       - django_media:/var/media
-    command: >
-      /bin/bash -c '
-      sleep 3 &&
-      python3 manage.py collectstatic --noinput &&
-      python3 manage.py migrate &&
-      uwsgi uwsgi.ini'
-    ports:
-      - "8087:8087"
-    links:
-      - db
-      - redis
-    env_file: .env
     environment:
+      DEBUG: 'True'
       DEV_ENV: 'True'
-      ODL_VIDEO_SERVICE_USE_WEBPACK_DEV_SERVER: 'True'
-      WEBPACK_DEV_SERVER_HOST: ${WEBPACK_DEV_SERVER_HOST:-localhost}
+      NODE_ENV: 'development'
       PORT: 8087
+      COVERAGE_DIR: htmlcov
+      DATABASE_URL: postgres://postgres@db:5432/postgres
+      ODL_VIDEO_USE_WEBPACK_DEV_SERVER: 'True'
```

Comment:
This is `ODL_VIDEO_SERVICE_USE_WEBPACK_DEV_SERVER` in the `settings.py`. Can you change one or the other to match?